docs/Basic-Guide.md

```diff
@@ -42,33 +42,42 @@ TensorFlow files in the Project window under **Assets** > **ML-Agents** >
 to the `Scripting Define Symbols` for each type of device you want to use 
 (**`PC, Mac and Linux Standalone`**, **`iOS`** or **`Android`**).
 
-![Project Settings](images/project-settings.png)
+   ![Project Settings](images/project-settings.png)
```

Comment:
What are these spaces for?

Reply:
makes the image aligned to the bullet points.  otherwise, it looks odd sitting off to the left

Reply:
Oh I see.